gems/aws-eventstream/lib/aws-eventstream/bytes_buffer.rb

```diff
@@ -57,7 +57,7 @@ def tell
       end
 
       def clear!
-        @data = ''
+        @data = ::String.new
```

Comment:
Should these be `String.new`? Unless I'm missing something, I'm not sure why these are prefixed with the empty namespace `::`. There are other places where this is changed too.

Reply:
This is my coding style.  
`::` prefix in ruby is an absolute way to address modules.

`String` can be relative.
~~~ruby
module ModuleA
  class String
  end
  class ClassB
    def initialize
      String.new # would refer to the class we just defined.
      ::String.new # is absolute, in this context it would be the same as ''.dup
    end
  end
end
~~~

In this pull request, I just want to be on the safe side, so I used absolute addressing.  
If this is against your style guides you can request a change.

Reply:
Ah yeah, I see. We generally prefer the relative styling if that's okay.